users/src/main/java/io/crate/operation/auth/HostBasedAuthentication.java

```diff
@@ -144,5 +147,10 @@ static boolean isValidAddress(Map.Entry<String, Map<String, String>> entry, Inet
             return false;
         }
 
+        static boolean isValidProtocol(Map.Entry<String, Map<String, String>> entry, String protocol) {
```

Comment:
I'd change the function signature so that the `entry.getValue().get()` call is done on callsite.